examples/timeseries/inject.py

```diff
@@ -41,7 +41,7 @@
 # Then we can download a simulation of the GW150914 signal from GWOSC:
 
 from astropy.utils.data import get_readable_fileobj
-url = ("https://www.gw-openscience.org/s/events/GW150914/P150914/"
+url = ("https://gwosc.org/s/events/GW150914/P150914/"
```

Comment:
Should this URL be world-readable?  Both the original and the modified URLs return `Forbidden` to me.

Reply:
I think it's just that this URL is out-of-date, and needs to be updated, I'll fix that later...